components/perf/dynamicRendering/src/.tern-port

```diff
@@ -1 +0,0 @@
-63476
```

Comment:
I guess this file should be gitignored

Reply:
The thing is already in the `.gitignore` file. I'm removing it and I expect to not see it again. :D